api/src/main/webapp/WEB-INF/cron_perf.yaml

```diff
@@ -3,3 +3,21 @@ cron:
   aou_skip_reason: "RW-6538"
 - url: /v1/cron/checkFreeTierBillingUsage
   aou_skip_reason: "RW-6280"
+- url: /v1/cron/checkRuntimes
+  aou_skip_reason: "RW-8945"
+- url: /v1/cron/checkPersistentDisks
+  aou_skip_reason: "RW-8945"
+- url: /v1/cron/updateGaugeMetrics
```

Comment:
path is wrong here: it's `/cron/monitoring/` (TODO: we should probably change this)